docs(concepts) a follow up on page creation (#3586)
webpack/webpack.js.org: src/content/concepts/under-the-hood.md

export default 'the app';
```

By using each other, the modules form a graph (`ModuleGraph`).

> Usually, modules don't exist by themselves and combine into chunks.

Author: combine into chunks doesnt make sense without changing to `are getting combined`, but i also feel we could add context and explicitly say that `When outputted by webpack, modules are usually combined into chunks.`

```suggestion
When outputted by webpack, modules are usually combined into chunks.
```

Reviewer: Maybe `During bundling, modules are combined into chunks.`?

Author: makes sense

Author: WDYT?
```suggestion
During the bundling process, modules are combined into chunks.
```

Reviewer: 👍
